app/student/text/StudentSubmission.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+$belowRoot = true;
+$isLoggedIn = true;
+$showNav = true;
+$displayClass =true;
+$isTeacher = true;
+
+$thisPage="Post Text Assignment";
+
+include('../../header.php');
+include('../../config.php');
```

Comment:
Hi @roxxid , everything above this line really doesn't need to be here...
The idea is that your "app" lives inside `text.php`, and you shouldn't navigate up a level

Following this pattern allows us to make changes to the portal without needing to make changes within other groups' sections

I showed Manlu--hope this makes sense!